core/cat/mad_hatter/plugin.py

```diff
@@ -48,15 +50,28 @@ def __init__(self, plugin_path: str, active: bool):
 
         # all plugins start active, they can be deactivated/reactivated from endpoint
         if active:
-            self.activate()
+            try:
```

Comment:
is this necessary? Having already a try/except in the `/plugins/toggle/` endpoint and inside `self._load_hooks_and_tools`?